### PR TITLE
chore: Leverages next-terraform-provider-release version

### PR DESCRIPTION
**Pull request by lantoli**
## Description

Leverages next-terraform-provider-release fix version.

All Done (Closed or Resolved) tickets are move from next-terraform-provider-release to the release version during the Release workflow.

Link to any related issue(s): CLOUDP-214955

## Type of change:

- [ ] Bug fix (non-breaking change which fixes an issue). Please, add the "bug" label to the PR.
- [ ] New feature (non-breaking change which adds functionality). Please, add the "enhancement" label to the PR. A migration guide must be created or updated if the new feature will go in a major version.
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected). Please, add the "breaking change" label to the PR. A migration guide must be created or updated.
- [ ] This change requires a documentation update
- [ ] Documentation fix/enhancement

## Required Checklist:

- [x] I have signed the [MongoDB CLA](https://www.mongodb.com/legal/contributor-agreement)
- [x] I have read the [contributing guides](https://github.com/mongodb/terraform-provider-mongodbatlas/blob/master/contributing/README.md)
- [x] I have checked that this change does not generate any credentials and that **they are NOT accidentally logged anywhere**.
- [x] I have added tests that prove my fix is effective or that my feature works per HashiCorp requirements
- [x] I have added any necessary documentation (if appropriate)
- [x] I have run make fmt and formatted my code
- [x] If changes include deprecations or removals, I defined an isolated PR with a relevant title as it will be used in the auto-generated changelog.
- [x] If changes include removal or addition of 3rd party GitHub actions, I updated our internal document. Reach out to the APIx Integration slack channel to get access to the internal document.

## Further comments

Example of output:
```
Issues moved: CLOUDP-237012, CLOUDP-237011
Version released, please check all tickets are marked as done: https://jira.mongodb.org/projects/CLOUDP/versions/39057
```

### CODE REVIEW

File: tools/jira-release-version/main.go [hunk 1]
```diff
@@ -51,18 +53,59 @@ func getVersion() string {
 }
 
 func getVersionID(client *jira.Client, versionName string) string {
```
Comment (AgustinBettati) [inline tools/jira-release-version/main.go]:
```suggestion
func getOrCreateVersionID(client *jira.Client, versionName string) string {
```

File: tools/jira-release-version/main.go [hunk 2]
```diff
@@ -13,19 +13,21 @@ import (
...
-	versionPrefix = "terraform-provider-"
+	envJira         = "JIRA_API_TOKEN"
+	envVersion      = "VERSION_NUMBER"
+	projectKey      = "CLOUDP"
+	jiraURL         = "https://jira.mongodb.org"
+	versionPrefix   = "terraform-provider-"
+	versionNameNext = "next-terraform-provider-release"
 )
 
 func main() {
```
Comment (AgustinBettati) [inline tools/jira-release-version/main.go:24]: [q] Curious if we were able to test this in any way, or if we will wait for a proper release.

**Review by AgustinBettati — APPROVED** [review]
LGTM

File: tools/jira-release-version/main.go (hunk 2 above)
Comment (lantoli) [reply tools/jira-release-version/main.go:24]: yes, i tested with a fake version

Comment (lantoli) [reply tools/jira-release-version/main.go:24]: adding an example of output in PR description

File: tools/jira-release-version/main.go (hunk 1 above)
Comment (lantoli) [reply tools/jira-release-version/main.go]: changed, thx

**Pull request merged**